### ISSUE TITLE
Could not find lwjgl-3.2.3-natives-${platform}.jar (org.lwjgl:lwjgl:3.2.3)

### ISSUE THREAD

**Issue by DeflatedPickle** [LWJGLX/lwjgl3-awt#25]
Whenever I try to use this library, in any project, I get this issue. It only occurs when I add this library. I'm sure I'm doing something wrong, I don't fully understand Gradle and I can't find any examples for this using it

`gradle.properties`
```
lwjglVersion=3.2.3
lwjglAwtVersion=0.1.7
```
`build.gradle`
```groovy
repositories {
    jcenter()
    mavenCentral()

    maven { url 'https://jitpack.io' }
}

dependencies {
    implementation "org.lwjgl:lwjgl:${lwjglVersion}"

    runtimeOnly "org.lwjgl:lwjgl:${lwjglVersion}:natives-windows-x86"
    runtimeOnly "org.lwjgl:lwjgl:${lwjglVersion}:natives-linux"
    runtimeOnly "org.lwjgl:lwjgl:${lwjglVersion}:natives-macos"

    implementation "org.lwjglx:lwjgl3-awt:${lwjglAwtVersion}"
}
```

Log:
```
Could not find lwjgl-3.2.3-natives-${platform}.jar (org.lwjgl:lwjgl:3.2.3).
Searched in the following locations:
    https://jcenter.bintray.com/org/lwjgl/lwjgl/3.2.3/lwjgl-3.2.3-natives-${platform}.jar

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

**Comment by tisoft** [comment]
I have added a fix for this in #26. Until then you can exclude the dependencies:

```
 dependencies {
     implementation "org.lwjgl:lwjgl:${lwjglVersion}"
 
     runtimeOnly "org.lwjgl:lwjgl:${lwjglVersion}:natives-windows-x86"
     runtimeOnly "org.lwjgl:lwjgl:${lwjglVersion}:natives-linux"
     runtimeOnly "org.lwjgl:lwjgl:${lwjglVersion}:natives-macos"
 
     implementation ("org.lwjglx:lwjgl3-awt:${lwjglAwtVersion}") {
        exclude group: '*', module: '*'
    }
    implementation "org.lwjgl:lwjgl-jawt:${lwjglVersion}"
 }
```

**Comment by DeflatedPickle** [comment]
Thanks for the response! I've added that to my `build.gradle`, but it isn't added to my external libraries after running the `dependency` task and refreshing IDEA

It's added to the submodule's dependencies, but causes the submodule to be underlined with red, with no further information that I can find
![image](https://user-images.githubusercontent.com/24986480/97063791-ac4f7480-1599-11eb-8d97-063c6de34b4c.png)
But it doesn't show up in the external libraries
![image](https://user-images.githubusercontent.com/24986480/97063533-2ed73480-1598-11eb-831a-dc3ce58b6cba.png)

Maybe this is because it's in a submodule?
